dashboard/app/models/stage.rb

```diff
@@ -46,6 +46,12 @@ def unplugged?
     script_levels.first.level.unplugged?
   end
 
+  def spelling_bee?
+    script_levels = script.script_levels.select{|sl| sl.stage_id == id}
+    return false unless script_levels.first
+    script_levels.first.level.spelling_bee?
```

Comment:
Mind using `script_levels.first.oldest_active_level.spelling_bee?` here? I'm (eventually) getting rid of `ScriptLevel.level`.